packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -44,240 +53,184 @@ export default {
+export const ActiveStep = {
+  name: 'Active Step',
+  render: () => html`
+    <div class="space-y-5">
+      <sd-step-group size="lg" orientation="horizontal" active-step="0">
+        <sd-step size="lg" orientation="horizontal" state="default">
+          <p slot="label">Label</p>
         </sd-step>
 
-        <sd-step orientation="horizontal">
-          <p slot="label">Exercitation ullamco laboris</p>
-          <sd-icon class="text-[48px]" name="content/image" slot="circle-content"></sd-icon>
+        <sd-step size="lg" orientation="horizontal" state="current">
+          <p slot="label">Label</p>
         </sd-step>
 
-        <sd-step orientation="horizontal">
-          <p slot="label">Reprehenderit qui in e name</p>
-          <sd-icon class="text-[48px]" name="content/image" slot="circle-content"></sd-icon>
+        <sd-step size="lg" orientation="horizontal" state="disabled">
+          <p slot="label">Label</p>
         </sd-step>
       </sd-step-group>
-      <div class="headline">size = 'sm'</div>
-      <sd-step-group size="sm" orientation="horizontal" not-interactive>
-        <sd-step orientation="horizontal">
-          <p slot="label">Lorem ipsum dolor sit</p>
-          <sd-icon class="text-3xl" name="content/image" slot="circle-content"></sd-icon>
+
+      <div class="flex gap-12 h-[20vh]">
+        <sd-step-group size="lg" orientation="vertical" active-step="1">
+          <sd-step size="lg" orientation="vertical" state="default">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="lg" orientation="vertical" state="current">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="lg" orientation="vertical" state="disabled">
+            <p slot="label">Label</p>
+          </sd-step>
+        </sd-step-group>
+      </div>
+    </div>
+  `
+};
+
+/**
+ * Use the `not-interactive` attribute to create a non-interactive step group.
+ */
+
+export const notInteractive = {
+  name: 'Not Interactive',
+  render: () => html`
+    <div>
+      <sd-step-group size="lg" orientation="horizontal" active-step="0" not-interactive>
+        <sd-step size="lg" orientation="horizontal" state="default">
+          <div slot="label">Label</div>
         </sd-step>
 
-        <sd-step orientation="horizontal">
-          <p slot="label">Exercitation ullamco laboris</p>
-          <sd-icon class="text-3xl" name="content/image" slot="circle-content"></sd-icon>
+        <sd-step size="lg" orientation="horizontal" state="current">
+          <span slot="label">Label</span>
         </sd-step>
 
-        <sd-step orientation="horizontal">
-          <p slot="label">Reprehenderit qui in e name</p>
-          <sd-icon class="text-3xl" name="content/image" slot="circle-content"></sd-icon>
+        <sd-step size="lg" orientation="horizontal" state="disabled">
+          <span slot="label">Label</span>
         </sd-step>
       </sd-step-group>
-    `;
-  }
+    </div>
+  `
 };
```

Comment:
How could we show that instead of numbers non-interactive could also contain a content icon?

![image](https://github.com/user-attachments/assets/211b4a19-24b1-4ad6-8a05-158e2806334b)

Reply:
I've created the icon story